tests/ui/linkage-attr/raw-dylib/elf/malformed-link-name.rs

```diff
@@ -0,0 +1,20 @@
+//@ only-x86_64-unknown-linux-gnu
```

Comment:
This can be `//@ only-elf`. The test should work fine on any ELF based target as codegen will never be hit anyway.